stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/AttachmentsPicker.kt

```diff
@@ -353,8 +356,15 @@ private fun AttachmentPickerOptions(
             enabled = hasPickedFiles || hasPickedImages,
             onClick = onSendAttachmentsClick,
             content = {
+                val layoutDirection = LocalLayoutDirection.current
+
                 Icon(
-                    modifier = Modifier.weight(1f),
+                    modifier = Modifier
+                        .weight(1f)
+                        .scale(
```

Comment:
Can we extract this somehow, e.g.:

```
public fun Modifier.mirrorRtl(layoutDirection: LayoutDirection): Modifier {
    return scale(
        scaleX = if (layoutDirection == LayoutDirection.Ltr) 1f else -1f,
        scaleY = 1f
    )
}


modifier = Modifier.mirrorRtl(LocalLayoutDirection.current)
```

Reply:
Yeah this is a good idea - I'll create a util!